extensions/amp-subscriptions/0.1/subscription-platform.js

```diff
@@ -26,12 +26,25 @@ import {PageConfig} from '../../../third_party/subscriptions-project/config';
  */
 export class SubscriptionPlatform {
 
+  /**
+   * Returns the service Id.
+   */
+  getServiceId() {
+  }
+
   /**
    * Requests entitlement for a subscription platform.
    * @return {!Promise<!Entitlements>}
    */
   getEntitlements() {
   }
+
+  /**
+   * Renders the UI specific to the platform
+   */
+  render() {
```

Comment:
Implement right away on google subscriptions: even if it's just an empty method. But I feel like the name is not broad enough. Maybe `activate()`?

Reply:
Done